backend/src/main/java/de/neuefische/teddyscoronadiaries/openweatherapi/config/OpenWeatherKeyConfig.java

```diff
@@ -0,0 +1,17 @@
+package de.neuefische.teddyscoronadiaries.openweatherapi.config;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Configuration
+@ConfigurationProperties(prefix = "openweather.key")
+public class OpenWeatherKeyConfig {
+
+    private String apiKey;
```

Comment:
Resulting key name is `openweather.key.apiKey` why not simply `openweather.key`?